_posts/installing_alpine.md

```diff
@@ -0,0 +1,152 @@
+
+![Terminal output](/img/posts/installing_alpine/13.png)
+
+![Terminal output](/img/posts/installing_alpine/14.png)
+
+Now let's add a user by typing in a lowercase login name. Name the user whatever you would like, but a a general convention is to keep it short and sweet. 
+
+Give this user a password as well.
+
+Select openssh as the ssh server.
+
+![Terminal output](/img/posts/installing_alpine/15.png)
+
+Now is the "scariest" part. Configuring and partitioning disks.
+
+To simply select an entire disk to wipe and install Alpine too, type that disks name and press enter.
+
+In a VM, that will USUALLY be "vda", but on bare metal it could be sda or sdb or nvme0n1 or anything else.
+
+This disk will be wiped and all data will be lost, so make sure you select the correct disk and proceed with caution.
+
+Select "sys" to install the operating system onto this disk, in a simple way without disk encryption. See "crypt" to install Alpine with encryption.
+
+![Terminal output](/img/posts/installing_alpine/16.png)
+
+Press "y" to proceed with wiping this disk and installing Alpine.
+
+Installing...
+
+![Terminal output](/img/posts/installing_alpine/17.png)
+
+When the installation is finished, the setup-alpine command will terminate and you will be back at your root shell. But we are still booted from the installation medium. So, we need to 
+
+Run the *reboot* command, and unplug the installation medium drive if you are on bare metal. For VM users, you don't have to do anything, as virt-manager will never mount this installation ISO again. 
+
+![Terminal output](/img/posts/installing_alpine/18.png)
+
+7. ## WELCOME TO ALPINE
+
+When prompted, log in with the user that we created earlier.
+This should launch the user's shell. Alpine is now installed, and now that you're logged in, this tutorial is done!
+
+You can do whatever you'd like. Install and setup any programs as you wish, or keep following around with this tutorial for some quick tips and tweaks. 
+
+
+In the following photo, I'm enabling an additional apk repository to add support for more software. 
+
+I achieve this by using vi to edit my */etc/apk/repositories* config file. Uncomment (remove the #) for any line that ends with /alpine/vXX.X/community.
+
+```bash
+doas vi /etc/apk/repositories
+```
+
+![Terminal output](/img/posts/installing_alpine/19.png)
+
+Then, update the package cache and upgrade the system's packages with apk upgrade.
+
+```bash
+doas apk upgrade
+```
+
+Install packages that you would like with *apk add*.
+
+Example:
+
+```bash
+doas apk add neofetch
+```
+
+![Terminal output](/img/posts/installing_alpine/20.png)
+
+![Terminal output](/img/posts/installing_alpine/21.png)
+
+By default, Alpine uses ash instead of bash for the shell. Bash has slightly more features and is the linux standard, so you may want to enable it. 
```

Comment:
Bash isn't necessarily the "Linux (capitalize) standard" - just one of the GNU Coreutils and a very *popular* default shell